guides/common/modules/snip_syncable-exports.adoc

```diff
@@ -0,0 +1,7 @@
++
+If you want to create a syncable export, add `--format=syncable`.
+By default, {Project} creates an importable export.
++
```

Comment:
@chris1984 I am a bit unsure how to interpret the output from hammer --help:

````
$ hammer content-export incremental version --help
 --format ENUM                                  Export formats.Choose syncable if the exported content needs to be in a yum
                                                format. This option is only available for yum, file repositories. Choose
                                                importable if the importing server uses the same version  and exported content
                                                needs to be one of yum, file, ansible_collection, docker, deb repositories.
                                                Possible value(s): 'syncable', 'importable'
````

Can you please shed some light; maybe ACK my summary?

* syncable > provide from an arbitrary webserver over HTTP to another Foreman server? (basically a "custom CDN" as mention in [10.4. Exporting the library environment in a syncable format](https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Exporting_the_Library_Environment_in_a_Syncable_Format_content-management)
* importable > create an archive, transfer to disk of another Foreman server; then import?